Tools/CISupport/ews-build/steps.py

```diff
@@ -915,6 +915,40 @@ def hideStepIf(self, results, step):
         return results == SUCCESS
 
 
+class InstallHooks(steps.ShellSequence):
+    name = 'install-hooks'
+    flunkOnFailure = False
+    haltOnFailure = False
+
+    def __init__(self, **kwargs):
+        super().__init__(timeout=30, logEnviron=False, **kwargs)
+
+    def run(self):
+        commands = [
+            ['git', 'config', 'include.path', '../metadata/git_config_extension'], [
+                'python3', 'Tools/Scripts/git-webkit',
+                'install-hooks', 'pre-push',
+                '--mode', 'no-radar',
+            ]
+        ]
+
+        source = self.getProperty('github.head.repo.full_name', None)
+        project = self.getProperty('project', None)
+        if project in GITHUB_PROJECTS and source:
+            commands[-1] += ['--level', 'github.com:{}={}'.format(source, GITHUB_PROJECTS.index(project))]
```

Comment:
modifying last element (commands[-1]) seems error-prone as someone else can modify the commands list later on without realizing that last element is special.

Reply:
Good point, making variable names more explicit